code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -74,6 +84,32 @@
 	var/greeted = 0
 	var/current_pda_messaging = null
 
+		//Interaction
+	var/response_help   = "pets"
+	var/response_disarm = "shoves"
+	var/response_harm   = "kicks"
+	var/harm_intent_damage = 15//based on 100 health, which is probably too much for a pai to have
+
+/mob/living/silicon/pai/attack_hand(mob/living/carbon/human/M as mob)
+	..()
+
+	switch(M.a_intent)
+
+		if(I_HELP)
+			M.visible_message("\blue [M] [response_help] \the [src]")
+
+		if(I_DISARM)
+			M.visible_message("\blue [M] [response_disarm] \the [src]")
+			M.do_attack_animation(src)
+			//TODO: Push the mob away or something
+

```

Comment:
I didn't add grabbing due to the potential bugginess headache it could be. you can collapse and pickup a pai anyways, or scoop it up and hold it. grabbing it is kind of redundant